src/bloom/bloom.c

```diff
@@ -292,20 +290,20 @@ __wt_bloom_hash_get(WT_BLOOM *bloom, WT_BLOOM_HASH *bhash)
 		WT_ERR(c->get_value(c, &bit));
 
 		if (bit == 0) {
-			result = WT_NOTFOUND;
+			ret = WT_NOTFOUND;
 			break;
 		}
 	}
 	WT_ERR(c->reset(c));
```

Comment:
This call to `WT_ERR` is going to overwrite the saved `ret` value now, isn't it?

Reply:
@agorrod 
> This call to `WT_ERR` is going to overwrite the saved `ret` value now, isn't it?

Yes, you are correct -- I apologize for the noisy change.